addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -61,20 +61,20 @@
 
 <t t-name="website.DeletePageDialog">
     <WebsiteDialog title="title" close="props.close">
-        <p t-if="props.resIds.length > 1">Are you sure you want to delete those pages?</p>
-        <p t-else="">Are you sure you want to delete this page?</p>
-        <p class="text-warning">Don't forget to update all links referring to it.</p>
+        <p t-if="props.resIds.length > 1">Ready to make those web pages disappear into thin air? Poof it will be gone forever!</p>
+        <p t-else="">Ready to make this web page disappear into thin air? Poof it will be gone forever!</p>
```

Comment:
Don't use "poof," as it is also a British slur. Of course, no human would interpret that as the cursed word, right? The thing is, some people machine-translate Odoo for their own use. We already got a ticket from a disenchanted customer after he was greeted with a very bad homophobic slur in Odoo 17. We resolved to remove it at once. Please don't bring back this disastrous wording